src/style.css

```
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
 }
+
+html {
+  --active-link: #00acdc;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
```

```suggestion
  max-width: 1200px;
  margin: 0 auto;
```
It will help to center your header horizontally